pkg/apis/harvesterhci.io/v1beta1/addon.go

```diff
@@ -7,9 +7,11 @@ import (
 type AddonState string
 
 const (
-	AddonEnabled  AddonState = "AddonEnabled"
-	AddonDeployed AddonState = "AddonDeploySuccessful"
-	AddonFailed   AddonState = "AddonDeployFailed"
+	AddonEnabled         AddonState = "AddonEnabled"
+	AddonDeployed        AddonState = "AddonDeploySuccessful"
+	AddonFailed          AddonState = "AddonDeployFailed"
+	DisablingAddon       AddonState = "DisablingAddon"
```

Comment:
`DisablingAddon` -> `AddonDisabling` to keep format 

`AddonDisablingFailed` or `AddonDisableFailed` ?